app/helpers/orders_helper.rb

```diff
@@ -1,2 +1,85 @@
 module OrdersHelper
+  def frendly_shipped_status(shipped_status)
+    shipped_status ? "Shipped" : "Not Shipped"
+  end
+
+  def pending_orders
+    orders = 0
+    @order_items.each do |order_item|
```

Comment:
'''@order_items.orders''' returns an empty array, because you can only call '''.order''' on a single '''order_item''. We'll leave this code here for now and think about more concise methods in the future.